### ISSUE TITLE
Remote database url support?

### ISSUE THREAD

**Issue by ilovemysillybanana** [sous-chefs/rundeck#85]
From the rundeck docs [here](http://rundeck.org/2.6.6/administration/setting-up-an-rdb-datasource.html)
We can see that rundeck supports specifying an external db source url as seen in this snippet of code:

```
dataSource.url = jdbc:oracle:thin:@localhost:1521:XE
dataSource.driverClassName = oracle.jdbc.driver.OracleDriver
dataSource.username = dbuser
dataSource.password = dbpass
dataSource.dialect = org.hibernate.dialect.Oracle10gDialect
```

This is how I setup the code:

```
# External Database properties
default['rundeck']['rdbms']['enable'] = 'true'
default['rundeck']['rdbms']['type'] = 'oracle'
default['rundeck']['rdbms']['location'] = 'xxx.xxx.xxx.xxx'
default['rundeck']['rdbms']['dbname'] = 'dbname'
default['rundeck']['rdbms']['dbuser'] = 'user'
default['rundeck']['rdbms']['dbpassword'] = 'password'
default['rundeck']['rdbms']['dialect'] = 'Oracle10gDialect'
default['rundeck']['rdbms']['port'] = 'xxxx'
```

When I check the file /etc/rundeck/rundeck-config.properties it looks like most of the settings are there except for one

```
dataSource.url = jdbc:oracle:thin:@:xxx.xxx.xxx.xxx:PORT:XE
dataSource.driverClassName = oracle.jdbc.driver.OracleDriver
dataSource.dialect = org.hibernate.dialect.Oracle10gDialect
dataSource.dbCreate=update
dataSource.username =user
dataSource.password =password
```

I need to have the XE part at the end changed, but it seems to always attach that no matter what I do and the docs make no mention of how that would be changed.

**Comment by ilovemysillybanana** [comment]
I found where to change the setting, value is hardcoded for some reason in rundeck-config.properties.erb.

**Comment by ARentz07** [comment]
Any thoughts on making this a node attribute that we can set?

**Comment by pcross616** [comment]
I agree, we should change this to a node attribute.

**Comment by ARentz07** [comment]
I'm pretty sure this is already supported by node attributes we expose: https://github.com/Webtrends/rundeck/blob/master/attributes/default.rb#L95-103

**Comment by atheiman** [comment]
It looks like this is hardcoded here https://github.com/Webtrends/rundeck/blob/v4.0.3/templates/default/rundeck-config.properties.erb#L12

This could be fixed with #127 tho
